steamwrap/api/Steam.hx

```diff
@@ -10,6 +10,15 @@ private enum LeaderboardOp
 	DOWNLOAD(id:String);
 }
 
+@:enum
+abstract SteamNotificationPosition(Int) to Int 
+{
+	var TopLeft = 0;
+	var TopRight = 1;
+	var BottomRight = 2;
+	var BottomLeft = 3;
+}
+
 typedef ControllerHandle = Int64;
```

Comment:
Nice! Thanks for using an enum on the haxe side too, so we can keep the user from dealing with "magic numbers"